ui/templates/base.html

```diff
@@ -29,6 +29,9 @@
     })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
     ga('create', '{{ GA_TRACKING_ID }}', 'auto');
+    if('{{ tracking_id }}'){
+      ga('create', '{{ tracking_id }}', 'auto');
```

Comment:
It looks like this takes a fourth parameter which is required when creating multiple trackers. Can you create some name including the program id for that parameter? https://developers.google.com/analytics/devguides/collection/analyticsjs/creating-trackers#working_with_multiple_trackers